BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -219,6 +228,10 @@ public class Plugin : IPlugin {
 				gameStatus.modNoEnergy = mainSetupData.gameplayOptions.noEnergy;
 				gameStatus.modMirror = mainSetupData.gameplayOptions.mirror;
 
+				if (gameStatus.modNoEnergy)	{
```

Comment:
Why is there a tab in here?

```suggestion
				if (gameStatus.modNoEnergy) {
```

Reply:
I don't know...